libs/filer/fake_filer_test.go

```diff
@@ -0,0 +1,109 @@
+package filer
+
+import (
+	"context"
+	"io"
+	"io/fs"
+	"testing"
+
+	"github.com/databricks/cli/libs/fakefs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFakeFiler_Read(t *testing.T) {
+	f := NewFakeFiler(map[string]fakefs.FileInfo{
+		"file": {},
+	})
+
+	ctx := context.Background()
+	r, err := f.Read(ctx, "file")
+	if !assert.NoError(t, err) {
```

Comment:
suggestion: `require.NoError` might be simpler and easier to read. Same for other instances below.